plugins/CoreHome/angularjs/reporting-menu/reportingmenu-model.js

```diff
@@ -161,5 +163,38 @@
                 return model.menu;
             });
         }
+
+        function fetchVisits(date, period, callback) {
+            var ajaxRequest = new ajaxHelper();
+
+            ajaxRequest.addParams({
+                module: 'API',
+                method: 'VisitsSummary.getVisits',
+                format: 'json',
+                date: date,
+                period: period,
+            }, 'get');
+            ajaxRequest.setCallback(callback);
+            ajaxRequest.setFormat('json');
+            ajaxRequest.withTokenInUrl();
+
+            ajaxRequest.send();
+        }
+
+        function fetchLiveVisits(date, period, callback) {
+            var ajaxRequest = new ajaxHelper();
+            ajaxRequest.addParams({
+                module: 'API',
+                method: 'Live.getLastVisitsDetails',
+                format: 'json',
+                date: decodeURIComponent(date),
+                period: period,
```

Comment:
I would suggest to add a `limit: 1` to that query. It's actually only relevant if there is a visit. No need to query all of them. So maybe the method should then be called something like `doesPeriodHasLiveVisits`

Reply:
thanks, make sense. I'll do it